content/cloud/user-settings.md

```diff
@@ -25,9 +25,9 @@ See the relevant sections below, for instructions on how to perform these action
 
 #### Create a PAT
 
-[Personal access tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) are an alternative to using passwords for authentication to GitHub when using the GitHub API or the command line. 
+Personal access tokens (PAT) enable continuous integration workflows by providing an alternative to using the standard login procedure which requires you to go through Fermyon Cloud to activate the device code
```

Comment:
@itowlson - I'd greatly appreciate your review on this re-word! Rajat and I were going back and forth brainstorming a simple but accurate way to describe the benefits of Fermyon PAT.